about-the-apprentices/2022/Bennett.md

```diff
@@ -0,0 +1,13 @@
+# Hello there
+
+Hello everyone! My name is `Bennett` and I'm a new apprentice at Sparkbox! Right now, I'm working remotely from `Minnesota`, which means I'm on `Central` time and my preferred pronouns are `He/Him`.
```

Comment:
The readme mentioned that we could delete the backticks! I vote that you can get rid of all of those throughout this doc.

Reply:
Ok made those changes.